### DISCUSSION TITLE
Opciones de pago: Culqi todavía no es compatible con este bloque

### DISCUSSION THREAD

**Discussion by alandelgadoalbines** [woocommerce/woocommerce#47165] [Checkout Flow]
Culqi todavía no es compatible con este bloque (Wodpress 6.5.2)
https://wordpress.org/plugins/culqi-checkout/
[https://s3.amazonaws.com/i.snag.gy/KpeuVR.jpg]
![KpeuVR](https://github.com/woocommerce/woocommerce/assets/137252220/4afc75db-b3be-4348-b896-50f7f6911632)

**Comment by senadir** [comment]
Culqi has to declare compatibility with Checkout block for it to work.
